src/components/Navbar/index.js

```diff
@@ -29,17 +29,19 @@ const Navbar = () => {
   return (
     <nav
       className={`nav-wrapper flex ${
-        pathArray.some((item) => currentPath.pathname === item)
+        pathArray.some((item) => currentPath.pathname === item) || token
           ? "jc-center"
           : "jc-between ai-center"
       } flex-gap`}>
       <div className='logo flex jc-between ai-center flex-gap'>
-        <NavLink to='/'>
-          <button className='btn btn-icon'>
-            <img src={devLogo} alt='dev_logo' />
-          </button>
-        </NavLink>
-        <h1 className='logo-text'>DEVSTORE</h1>
+        <div className='flex flex-gap jc-center  ai-center'>
+          <NavLink to='/'>
+            <button className='btn btn-icon'>
+              <img src={devLogo} alt='dev_logo' />
+            </button>
+          </NavLink>
```

Comment:
If you arent using style classes Link would be suffice too

Reply:
yeah you can use any of the link or NavLink as the name suggest NavLink so I am using it for Navbar.

Reply:
I have not added the category section yet so yeah it will not work now.

Reply:
Search Component Will work if you are not login.